packages/lodestar/src/network/reqResp.ts

```diff
@@ -131,7 +132,7 @@ export class ReqResp extends (EventEmitter as IReqEventEmitterClass) implements
       id,
       err,
       (async function* () {
-        if (response !== null && response !== undefined) {
+        if (notNullish(response)) {
```

Comment:
I created `notNullish` to filter items in an array that contain nullish items. IMO it's best to just use the original form because it's more obvious and transparent. Future code readers now have to worry about what is `notNullish` actually doing.

Reply:
gotcha.  i replaced the places where i used notNullish with isNil from lodash instead